core/client/src/client.rs

```diff
@@ -350,6 +350,19 @@ impl<B, E, Block, RA> Client<B, E, Block, RA> where
 			.map(StorageData))
 	}
 
+	/// Return a child storage entry keys in state in a block of given hash with given prefix.
```

Comment:
Nit: I think "a block of given hash" is a bit inaccurate: we use `BlockId` here which can be not only a number. If you are going to fix this, please fix `child_storage` as well.